pandas/tests/indexes/common.py

```diff
@@ -325,6 +372,11 @@ def test_argsort(self):
 
     def test_numpy_argsort(self):
         for k, ind in self.indices.items():
+
+            # 'mixedIndex' unorderable in Python3
+            if k in ['mixedIndex']:
```

Comment:
I rather wanted to skip this test because it raises only in python 3. But if you think it's worth adding, then I'll do it.